src/containers/Tasks/ImportAttestation/ImportAttestation.tsx

```diff
@@ -49,30 +49,29 @@ const ImportAttestation: React.FC<Props> = ({
 
   useEffect(() => {
     claims.forEach(claim => {
-      let newRequestForAttestation
-      let newClaimId
       claim.requestForAttestations.forEach(
         ({ requestForAttestation }): void => {
           if (requestForAttestation.rootHash === attestation.claimHash) {
-            newRequestForAttestation = requestForAttestation
-            newClaimId = claim.id
+            setRequestForAttestation(requestForAttestation)
+            setClaimId(claim.id)
           }
         }
       )
+    })
+  }, [claims, attestation])
 
-      if (!newRequestForAttestation) {
+  useEffect(() => {
+    claims.forEach(claim => {
+      if (!requestForAttestationEntry) {
         claim.attestedClaims.forEach(({ request }) => {
           if (request.rootHash === attestation.claimHash) {
-            newRequestForAttestation = request
-            newClaimId = claim.id
+            setRequestForAttestation(request)
+            setClaimId(claim.id)
           }
         })
       }
-
-      setRequestForAttestation(newRequestForAttestation)
-      setClaimId(newClaimId)
     })
-  }, [claims, attestation])
```

Comment:
Why these changes in this file?

Reply:
The attestation was being returned as `undefined` when importing. The data was there but not being assigned.

Reply:
we talked about it and reverted it